### DISCUSSION TITLE
Question about album art URL

### DISCUSSION THREAD

**Discussion by lotussutol** [AzuraCast/AzuraCast#6171] [Q&A]
I am a newb...so sorry if this is a dumb question.
I use radioDJ/rocket broadcaster to transmit a stream to azuracast.
I get a lot of "No Album Art"

I see sometimes the URL for the album art is something like this.
https://radio.x-factorradio.com/static/uploads/album_art.1656737058.jpg (My server/no cover art)
But sometimes it shows the album art and is using...and uses the cover art archive url.
https://coverartarchive.org/release/02c5efaa-1c03-4929-99d4-49f6ba4877f3/22723672103-1200.jpg

What decides which gets used? Does it do a check with the external URL first and if nothing is found uses the internal no cover found URL?

IF I upload the same songs to azuracast and let azuracast play the music it almost shows the proper album art all the time...but that is not a solution for me. It was just to test.

I am looking to see how I can improve my percent of no album art ratio.

Thanks,
I did a search for similar terms and found some close matches but not the same exact problem. Mostly no art at all via stream problems.

**Comment by BusterNeece** [comment]
Here's the current order we use to determine which album art to show:

```
For album art resolution, the order now goes:
 - Album art on the file itself (which is set from cover art)
 - Remote album art if available
 - Default artwork of the current live streamer (if any)
 - Station uploaded custom art
 - Station custom album art URL
 - Global uploaded custom art
 - Global custom art URL
 - The default AzuraCast album art image
```
